node/src/components/linear_chain.rs

```diff
@@ -142,36 +140,6 @@ where
         event: Self::Event,
     ) -> Effects<Self::Event> {
         match event {
-            Event::Request(LinearChainRequest::BlockRequest(block_hash, sender)) => async move {
```

Comment:
I will probably learn it later in the review process but why are we removing these? It should be the `LinearChain` component that answers these queries about the _linear chain_ blocks.